### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.core.exceptions.AzureError.raise_with_traceback
  class: azure.core.exceptions.AzureError
  fullName: azure.core.exceptions.AzureError
  inheritance:
  - inheritance:
    - inheritance:
      - type: builtins.object
      type: builtins.BaseException
    type: builtins.Exception
  langs:
  - python
  module: azure.core.exceptions
  name: AzureError
  summary: Base exception for all errors.
  syntax:
    content: AzureError(message, *args, **kwargs)
  type: class
  uid: azure.core.exceptions.AzureError
- class: azure.core.exceptions.AzureError
  fullName: azure.core.exceptions.AzureError.raise_with_traceback
  langs:
  - python
  module: azure.core.exceptions
  name: raise_with_traceback()
  syntax:
    content: raise_with_traceback()
    parameters: []
  type: method
  uid: azure.core.exceptions.AzureError.raise_with_traceback
references:
- fullName: azure.core.exceptions.AzureError.raise_with_traceback
  isExternal: false
  name: raise_with_traceback()
  parent: azure.core.exceptions.AzureError
  uid: azure.core.exceptions.AzureError.raise_with_traceback
